core/events/utils.ts

```diff
@@ -290,6 +290,17 @@ export function filter(queueIn: Abstract[], forward: boolean): Abstract[] {
         lastEvent.newParentId = moveEvent.newParentId;
         lastEvent.newInputName = moveEvent.newInputName;
         lastEvent.newCoordinate = moveEvent.newCoordinate;
+        if (moveEvent.reason) {
+          if (lastEvent.reason) {
+            // Contatenate reasons into a comma separated string
```

Comment:
nit: Concatenate (if this comment still exists pending comment re: using an array)

Reply:
Done.